specification/ml/stop_data_frame_analytics/MlStopDataFrameAnalyticsRequest.ts

```diff
@@ -76,4 +76,36 @@ export interface Request extends RequestBase {
      */
     timeout?: Duration
   }
+  body?: {
+    /**
+     * If provided, must be the same identifier as in the path.
+     */
+    id?: Id
+    /**
+     * Specifies what to do when the request:
+     *
+     * 1. Contains wildcard expressions and there are no data frame analytics
+     * jobs that match.
+     * 2. Contains the _all string or no identifiers and there are no matches.
+     * 3. Contains wildcard expressions and there are only partial matches.
+     *
+     * The default value is true, which returns an empty data_frame_analytics
+     * array when there are no matches and the subset of results when there are
+     * partial matches. If this parameter is false, the request returns a 404
+     * status code when there are no matches or only partial matches.
+     * @server_default true
+     */
+    allow_no_match?: boolean
+    /**
+     * If true, the data frame analytics job is stopped forcefully.
+     * @server_default false
+     */
+    force?: boolean
+    /**
+     * Controls the amount of time to wait until the data frame analytics job
+     * stops. Defaults to 20 seconds.
+     * @server_default 20s
+     */
+    timeout?: Duration
+  }
```

Comment:
body is read here: https://github.com/elastic/elasticsearch/blob/4381faec9ab38dcca287e1d2d0e1da481afbe147/x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/dataframe/RestStopDataFrameAnalyticsAction.java#L41-L43

using this parser: https://github.com/elastic/elasticsearch/blob/4381faec9ab38dcca287e1d2d0e1da481afbe147/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StopDataFrameAnalyticsAction.java#L56-L61